### DISCUSSION TITLE
Realtime sync

### DISCUSSION THREAD

**Discussion by twoedges** [Mimetis/Dotmim.Sync#915] [Q&A]
Hi all,
it's possibile to have real-time sync between client&server , instead of force SynchronizeAsync every 5 minutes ?

**Comment by Mimetis** [comment]
nop

**Reply by twoedges** [reply]
hi,
thanks for your response

Qhat could be the best solution to keep client and server db synchronized while using the client program?
For instance, it's possible to run synchronize only when the data has changed perhaps by querying the tables created by DMS to manage the synchronization?

**Reply by Mimetis** [reply]
I don't know, honestly. 
Make a sql watcher on tracking tables, or kind of, could be a good idea, I guess.
